app/assets/javascripts/actions/notifications.js

```diff
@@ -0,0 +1,91 @@
+import actionTypes from './actionTypes';
+import status from 'http-status';
+
+export const addNotification = (notification) => ({
+  type: actionTypes.ADD_NOTIFICATION,
+  notification
+});
+
+export const removeNotification = (id) => ({
+  type: actionTypes.REMOVE_NOTIFICATION,
+  id
+});
+
+export const sendSuccessNotification = (message) =>
+  (dispatch, getState, { Notification }) => {
+    const newNotification = Notification.createNotification({
+      type: Notification.types.SUCCESS,
+      message
+    });
+
+    dispatch(addNotification(newNotification));
+
+    setTimeout(() => {
+      dispatch(removeNotification(newNotification.id));
+    }, 4000);
+  }
+
+export const sendErrorNotification = (error) =>
+  !!error.response
+    ? sendServerErrorNotification(error)
+    : sendDefaultErrorNotifiction()
```

Comment:
There is a typo here

```suggestion
    : sendDefaultErrorNotification()
```

Reply:
nice